PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-detail/pokemon-detail.component.ts

```diff
@@ -249,17 +293,35 @@ export class PokemonDetailComponent implements OnInit {
             });
     }
 
+    private convertDate(date: String) {
```

Comment:
I would move these generic helper classes to a separate class / date service + dependency injection